src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -13,26 +13,51 @@ Starting April 12, 2021, we're allowing some customers who have users on our ori
 
 ## Requirements [#requirements]
 
 The easiest way to see if you have access to the user migration wizard is to check if it’s visible [in the UI](#find).
 
 Factors affecting access to the migration wizard: 
 * You must have the [Owner role](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/#roles). 
-* The migration wizard is **not** available for some organizations that have a more complex account structure. If the migration wizard is not available and you wish to migrate your users, contact your New Relic account representative. 
+* The migration wizard is **not** available for some organizations with a more complex account structure. If the migration wizard is not available and you wish to migrate your users, contact your New Relic account representative. 
 
-## Before beginning [#before-beginning]
+## Before you start [#before-beginning]
 
 The more complex your organization is, the more it may help you to first understand our [new user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts). A brief summary: 
 * Users reside within a container known as an “authentication domain,” which governs how those users are added to New Relic (manually with the UI vs. automatically with a SCIM-based identity provider), and how they log in (manually with email/password vs. using SAML SSO), and other settings. Typically organizations have just one or two domains: one with the default manual settings and one for the more automatic methods.  
 * Users can be assigned to one or more groups (for example, our default **Admin** group or a custom group like **Contractors**). For large organizations, users are often in multiple groups. 
 * When you want to add new groups, or give groups access to more accounts, you must create an access grant. This assigns a group access to: 1) a specific role and 2) a specific account. For example, you may give a Contractors group access to our default **All product admin** role on one or more of your accounts, or give that group a custom role (a role being a custom set of [capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts)). 
 
-The more complex your organization is, the more it may help you to first plan out how your user groups and access grants will be structured. 
+To read more about these concepts, see [User management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
+
+
+## Think about what groups you'll need [#plan-groups]
+
+Once you have a decent understanding of the user management concepts, you'll want to think about how you'll implement groups for your organization. The more complex your organization is, the more it may help you to first plan out how your user groups and access grants will be structured. 
```

Comment:
How would someone measure their level of understanding? I worry that this might leave someone in a state of being uncertain.